module Main where
import CH9.AsyncExceptions

main :: IO ()
main = getURLsAsyncCancellable
